pennylane/qchem/molecule.py

```diff
@@ -42,7 +42,7 @@ class Molecule:
         charge (int): net charge of the molecule
         mult (int): Spin multiplicity :math:`\mathrm{mult}=N_\mathrm{unpaired} + 1` for
             :math:`N_\mathrm{unpaired}` unpaired electrons occupying the HF orbitals. Currently,
-            openshell systems are not supported; ``mult`` must be equal to :math:`1`.
+            open-shell systems are not supported; ``mult`` must be equal to :math:`1`.
```

Comment:
We should modify this as with this change we now allow constructing molecules for open-shell systems.

Reply:
Do we need a warning as we raise errors when open-shell systems are used with those function?

Reply:
I meant a warning card for the `docs` like the one we added in the `molecular_hamiltonian` regarding the deprecation.

Reply:
We can safely skip any checks for openshell system in the Molecule class, because it is now the general and central object that works with all backends. @ddhawan11 please remove the line saying: "Currently, openshell systems are not supported; ``mult`` must be equal to :math:`1`.